src/Tests/AggregatorCronTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\ultimate_cron\Tests\AggregatorCronTest.
+ */
+
+namespace Drupal\ultimate_cron\Tests;
+
+use Drupal\aggregator\Tests\AggregatorTestBase;
+
+/**
+ * Update feeds on cron.
+ *
+ * @group ultimate_cron
+ */
+class AggregatorCronTest extends AggregatorTestBase {
```

Comment:
Lets rename the test to somethign like CronQueueTest. Add a description that says that we are using aggregator module to test that queues are processed on cron.